portal/src/OnboardingRedirect.tsx

```diff
@@ -20,7 +20,7 @@ const OnboardingRedirect: React.VFC = function OnboardingRedirect() {
     if (apps && apps.length > 0) {
       navigate("/");
     } else {
-      navigate("/projects/create");
+      navigate("/onboarding-survey");
```

Comment:
You need to check the custom attribute to determine if the user needs to fill the survey or not.

Reply:
Should I create a new query in the backend as well? I think it is currently not possible to query the viewer's custom attributes currently in portal. On the other hand, if we keep the current implementation, the only possible issue is that the first-time user skips the onboarding survey by changing the URL to `.../projects/create`, which is highly unlikely.

Reply:
Actually I guess they can get out in the future if the 404 page (that doesnt exist currently) contains a link back to `/projects` page, and the user tried to skip the onboarding survey by typing random things into the URL, going to the 404 page.

Reply:
> Should I create a new query in the backend as well? I think it is currently not possible to query the viewer's custom attributes currently in portal

I have the following suggestion:

1. Modify the `Viewer` node of the portal api. Add one field `isOnboardingSurveyCompleted: boolean`.
2. In the resolver of `Viewer`, query the user's custom attribute. Check if the survey is completed by existence of that attribute, and compute the corresponding value of `isOnboardingSurveyCompleted`.
3. In frontend, query and check `isOnboardingSurveyCompleted` in the viewer api.

Reply:
Ooh ok thanks for your helpful suggestion, will follow